core/src/main/java/org/millr/slick/servlets/comment/EditComment.java

```diff
@@ -62,7 +62,10 @@
     DispatcherService dispatcherService;
     
     protected void doPost(SlingHttpServletRequest request, SlingHttpServletResponse response) throws ServletException, IOException{
-        LOGGER.info(">>>> Entering doPost");
+		request.setCharacterEncoding("UTF-8");
```

Comment:
Let's use `org.apache.commons.lang3.CharEncoding` here. Then you can import that and use...

`request.setCharacterEncoding(CharEncoding.UTF_8);`